pyproject.toml

```diff
@@ -106,6 +106,8 @@ base = [
 
 transformers = [
     "transformers==4.44.1",
+    "diffusers==0.30.0",
+    "torchmetrics[image]>=0.7.0"
```

Comment:
Is there no conflict with lightening torchmetric?

Reply:
This version specification is the same as in `pytorch_lightning`